tempto-core/src/main/java/com/teradata/tempto/internal/hadoop/hdfs/AbstractWebHdfsClient.java

```diff
@@ -188,13 +175,13 @@ public void loadFile(String path, String username, OutputStream outputStream)
 
     @Override
     @SuppressWarnings("unchecked")
-    public long getLength(String path, String username)
+    public long getLength(String path)
     {
-        HttpGet readRequest = new HttpGet(buildUri(path, username, "GETFILESTATUS"));
-        try (CloseableHttpResponse response = httpClient.execute(readRequest)) {
+        HttpGet readRequest = new HttpGet(buildUri(path, "GETFILESTATUS", ImmutableMap.of()));

```

Comment:
`ImmutableMap.of()` also return a static instance. But i agree that `emptyMap()` is more readable.